lib/src/user.dart

```diff
@@ -298,6 +308,23 @@ class ApiKey {
   int get hashCode => id.hashCode;
 }
 
+/// A class exposing functionality for calling remote Atlas Functions.
+class FunctionsClient {
+  final User _user;
+
+  FunctionsClient._(this._user);
+
+  /// Calls a remote function with the supplied arguments.
+  /// @name The name of the Atlas function to call.
+  /// @functionArgs - Arguments that will be sent to the Atlas function. They have to be json serializable values.
+  Future<dynamic> call(String name, [List<Object?>? functionArgs]) async {
```

Comment:
Should this be cancellable?

Reply:
It isn't cancellable in the rest of the SDKs.

Reply:
I don't think this is cancellable in the rest of the Realm SDKs?

Reply:
this does have a timeout though (as it should)

Reply:
But should it be? `Realm.open` supports cancel now. Same with `Realm.writeAsync`. This is a potentially long running operation?

Reply:
We have decided to align with the other SDKs for now. Hence no support. for cancellation.

Reply:
I think it's not cancelable due to an oversight. I don't see a strong reason not to have it be cancelable for dart if it's easy to do.

Reply:
Added a discussion point.